page-contact.php

```diff
@@ -35,7 +35,7 @@
 <?php if( have_rows('board_members') ): ?>
 	<section class="board-members contact--page__container" id="board">
 		<h2 class="board-members__title contact--page__blocktitle">
-			Send one of our board members an email
+			Send one of our confidential mentors an email
```

Comment:
Oké nice, in de basis wel wat er moet gebeuren, maar als je dit doet wordt het dat ook op https://studieverenigingid.nl/contact/, terwijl je het daar wel zo wil houden. De twee makkelijkste opties:
1. Dit veranderen naar ‘one of us’
2. Een if statement erin zetten om te kijken of het de contactpagina is (-> board members) of confidential mentor, met weer een paar opties:
    1. Zoals hier, door te kijken of het /x/ is of /x/y/ https://github.com/studieverenigingid/i.d-Website/blob/main/page-member.php#L18
    2. Of op basis van de slug https://github.com/studieverenigingid/i.d-Website/blob/main/functions.php#L135
    3. Op basis van de titel
Ze hebben allemaal nadelen 😬 En misschien is er nog een goede optie die ik heb gemist!